plugin/backend.go

```diff
@@ -120,6 +121,16 @@ func (pm *Manager) Push(name string, metaHeader http.Header, authConfig *types.A
 	}
 	defer config.Close()
 
+	var dummy map[string]interface{}
+	err = json.NewDecoder(config).Decode(&dummy)
```

Comment:
This doesn't catch garbage after the first JSON object. Like `{}{}`. As we don't actually use the reader anymore, it is probably easier to just read the whole file, unmarshal and then wrap it with `bytes.Buffer` before calling `Push`.